package.json

```diff
@@ -41,8 +41,8 @@
     "tslint": "4.1.1",
     "tslint-loader": "3.3.0",
     "typescript": "2.1.4",
-    "webpack": "1.14.0",
-    "webpack-dev-server": "1.16.2"
+    "webpack": "2.1.0-beta.28",
```

Comment:
@Hotekk, I was going by the migrate to webpack 2 docs. I'll upgrade to the RC versions.